OpenMEEG/libs/OpenMEEG/include/assemble.h

```diff
@@ -51,85 +51,85 @@ namespace OpenMEEG {
         virtual ~Head2EEGMat () {};
     };
 
     class OPENMEEG_EXPORT Head2ECoGMat: public virtual SparseMatrix {
     public:
-        Head2ECoGMat (const Geometry& geo, const Sensors& electrodes, const Interface& i);
-        Head2ECoGMat (const Geometry& geo, const Sensors& electrodes, const std::string& id); // mainly for SWIG
+        Head2ECoGMat(const Geometry& geo,const Sensors& electrodes,const Interface& i);
+        Head2ECoGMat(const Geometry& geo,const Sensors& electrodes,const std::string& id); // mainly for SWIG
         virtual ~Head2ECoGMat () {};
     };
 
     class OPENMEEG_EXPORT Head2MEGMat: public virtual Matrix {
     public:
-        Head2MEGMat (const Geometry& geo, const Sensors& sensors);
+        Head2MEGMat (const Geometry& geo,const Sensors& sensors);
         virtual ~Head2MEGMat () {};
     };
 
     class OPENMEEG_EXPORT SurfSource2MEGMat: public virtual Matrix {
     public:
-        SurfSource2MEGMat (const Mesh& sources, const Sensors& sensors);
+        SurfSource2MEGMat (const Mesh& sources,const Sensors& sensors);
         virtual ~SurfSource2MEGMat () {};
     };
 
     class OPENMEEG_EXPORT DipSource2MEGMat: public virtual Matrix {
     public:
-        DipSource2MEGMat(const Matrix& dipoles, const Sensors& sensors);
+        DipSource2MEGMat(const Matrix& dipoles,const Sensors& sensors);
         virtual ~DipSource2MEGMat () {};
     };
 
     class OPENMEEG_EXPORT DipSource2InternalPotMat: public virtual Matrix {
     public:
-        DipSource2InternalPotMat(const Geometry& geo, const Matrix& dipoles,
-                                 const Matrix& points, const std::string& domain_name = "");
+        DipSource2InternalPotMat(const Geometry& geo,const Matrix& dipoles,
+                                 const Matrix& points,const std::string& domain_name = "");
         virtual ~DipSource2InternalPotMat () {};
     };
 
     class OPENMEEG_EXPORT CorticalMat: public virtual Matrix {
     public:
-        CorticalMat (const Geometry& geo, const Head2EEGMat& M, const std::string& domain_name = "CORTEX",
-                const unsigned gauss_order=3, double alpha=-1., double beta=-1., const std::string &filename="");
+        CorticalMat (const Geometry& geo,const Head2EEGMat& M,const std::string& domain_name = "CORTEX",
+                const unsigned gauss_order=3,double alpha=-1.,double beta=-1.,const std::string &filename="");
         virtual ~CorticalMat () {};
     };
 
     class OPENMEEG_EXPORT CorticalMat2: public virtual Matrix {
     public:
-        CorticalMat2(const Geometry& geo, const Head2EEGMat& M, const std::string& domain_name = "CORTEX",
-                const unsigned gauss_order=3, double gamma=1., const std::string &filename="");
+        CorticalMat2(const Geometry& geo,const Head2EEGMat& M,const std::string& domain_name = "CORTEX",
+                     const unsigned gauss_order=3,double gamma=1.,const std::string &filename="");
```

Comment:
@papadop we really need a style checker on travis as otherwise I'll keep undoing your styles changes. I put spaces after , ...